### PR TITLE
Added symbols icon and fraction key to second page

### PR DESCRIPTION
**Pull request by tomkuschka**
## Description

This PR adds a new icon for the symbols page and a fraction button to the second page of the keyboard

| New icon for symbols | Fraction button |
|---|---|
|image|image|

## Checklist

*Remove `If [...]` items that do not apply to your PR.*

- [x] I have made myself familiar with the CaTeX
      [contributing guide](https://github.com/simpleclub/math_keyboard/blob/master/CONTRIBUTING.md).
- [x] I added a PR description.
- [x] If this PR changes anything about the main `math_keyboard` or `example` package
      (also README etc.), I created an entry in `CHANGELOG.md` (`## UPCOMING RELEASE` if the change
      on its own is not worth an update).
- [x] If this PR includes a notable change in the `math_keyboard` package, I updated the version
        according to [Dart's semantic versioning](https://stackoverflow.com/questions/66201337/how-do-dart-package-versions-work-how-should-i-version-my-flutter-plugins/66201338#66201338).
- [ ] All required checks pass.

### CODE REVIEW

File: math_keyboard/lib/src/widgets/math_keyboard.dart [hunk 1]
```diff
@@ -341,7 +342,8 @@ class _Buttons extends StatelessWidget {
                         else if (config is PageButtonConfig)
                           _BasicButton(
                             flex: config.flex,
-                            label: controller.secondPage ? '123' : 'f(x)',
+                            icon: CustomKeyIcons.key_symbols,
+                            // label: controller.secondPage ? '123' : 'f(x)',
```
Comment (edhom) [inline math_keyboard/lib/src/widgets/math_keyboard.dart:346]:
```suggestion
```

**Review by edhom — APPROVED** [review]
lgtm

**Review by edhom — DISMISSED** [review]

**Review by ukasz123 — DISMISSED** [review]

**Review by creativecreatorormaybenot — CHANGES_REQUESTED** [review]
Otherwise, LGTM 🔥🔥

**Review by creativecreatorormaybenot — DISMISSED** [review]
Do not want to delay you :P

Feel free to adjust :)

**Comment by creativecreatorormaybenot** [comment]
(but would be cool if you replace the accent color usage by color scheme secondary :D)

**Review by edhom — APPROVED** [review]

**Review by creativecreatorormaybenot — APPROVED** [review]

**Pull request merged**